Source/Core/Core/PowerPC/JitArm64/JitArm64_RegCache.cpp

```diff
@@ -279,14 +281,25 @@ ARM64Reg Arm64FPRCache::R(u32 preg)
 	switch (reg.GetType())
 	{
 	case REG_REG: // already in a reg
+	case REG_LOWER_PAIR:
 		return reg.GetReg();
```

Comment:
Do we need to load the upper bits of only_lower is false, and reg.GetType is REG_LOWER_PAIR?